### ISSUE TITLE
"loss: nan" during re-training

### ISSUE THREAD

**Issue by Haibot-Lee** [BBillot/SynthSeg#63]
Hi,
I'm trying to run the training tutorial to fit my own training data. But the log shows that the loss is nan after number of iterations. Does this mean the model training is failed? Should I stop training and check my data? Thanks a lot!

example logs:
  43/5000 [..............................] - ETA: 11:03:45 - loss: 23.8461
  44/5000 [..............................] - ETA: 11:03:08 - loss: 23.8833
  45/5000 [..............................] - ETA: 11:02:39 - loss: 23.8257
  46/5000 [..............................] - ETA: 11:02:07 - loss: 23.8267
  47/5000 [..............................] - ETA: 11:01:44 - loss: 23.8493
  48/5000 [..............................] - ETA: 11:01:22 - loss: 23.8251
  49/5000 [..............................] - ETA: 11:01:10 - loss: nan    
  50/5000 [..............................] - ETA: 11:00:36 - loss: nan
  51/5000 [..............................] - ETA: 11:00:02 - loss: nan
  52/5000 [..............................] - ETA: 10:59:42 - loss: nan
  53/5000 [..............................] - ETA: 10:59:27 - loss: nan
  54/5000 [..............................] - ETA: 10:59:01 - loss: nan
  55/5000 [..............................] - ETA: 10:58:43 - loss: nan
  ...
  All of the loss is nan later on.

**Comment by BBillot** [comment]
Yep there's something wrong with your loss. So stop the training and check what's going on. 
If you're using your own training data, I'm guessing there's a problem with the lists of generation/segmentation labels or the gmm classes. check the tutorials for for info on this :)
Best
Benjamin

**Comment by BBillot** [comment]
closing this for now, since there is no activity
